apps/condo/domains/marketplace/schema/GetInvoiceByUserService.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Generated by `createservice marketplace.GetInvoiceByUserService '--type=queries'`
+ */
+
+const { get, isEmpty } = require('lodash')
+
+const { GQLCustomSchema } = require('@open-condo/keystone/schema')
+const { find } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/marketplace/access/GetInvoiceByUserService')
+const { MarketItem } = require('@condo/domains/marketplace/utils/serverSchema')
+
+
+const GetInvoiceByUserService = new GQLCustomSchema('GetInvoiceByUserService', {
+    types: [
+        {
+            access: true,
+            type: 'input GetInvoiceByUserInput { dv: Int!, sender: SenderFieldInput!, organization: OrganizationWhereUniqueInput!, property: PropertyWhereUniqueInput, ticketIds: [ID!] }',
+        },
+        {
+            access: true,
+            type: 'type MarketSkuInfo { sku: String! imageUrl: String! }',
+        },
+        {
+            access: true,
+            type: 'type GetInvoiceByUserOutput { invoices: [Invoice!]!, skuInfo: [MarketSkuInfo!]! }',
+        },
+    ],
+    
+    queries: [
+        {
+            access: access.canGetInvoiceByUser,
+            schema: 'executeGetInvoiceByUser (data: GetInvoiceByUserInput!): GetInvoiceByUserOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                const { data: { organization, property, ticketIds } } = args
+                const userId = get(context, ['authedItem', 'id'])
+                const propertyId = get(property, 'id')
+                const organizationId = get(organization, 'id')
+
+                const where = {
+                    deletedAt: null,
+                    client: { id: userId },
+                    context: { organization: { id: organizationId } },
+                }
+                if (propertyId) where.property = { id: propertyId }
+                if (!isEmpty(ticketIds)) where.ticket = { id_in: ticketIds }
+
+                const invoices = await find('Invoice', where)
+
+
+                const allSku = []
+                for (const invoice of invoices) {
+                    allSku.push(...invoice.rows.map(({ sku }) => sku))
+                }
+
+                const skuInfo = []
+                for (const sku of allSku) {
+                    const marketItem = await MarketItem.getOne(context, {
+                        sku,
+                        deletedAt: null,
+                        organization: { id: organizationId },
+                    })
```

Comment:
It's better to use one query instead of many
```js
where: {
  deletedAt: null,
  OR: [
    {sku, org},
    {sku, org},
    {sku, org},
  ],
}
```